msteams-platform/concepts/messaging-extensions/create-extensions.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Action based messaging extensions
+description: Describes how to develop action based messaging extensions
+keywords: teams messaging extensions messaging extensions search
+ms.date: 11/07/18
+---
+
+# Initiate actions with messaging extensions (Developer preview)
+
+Action based messaging extensions allow your users to trigger actions in 3rd party services while inside of Teams.
+
+![Example of messaging extension card](~/assets/images/compose-extensions/ceexample.png)
+
+The following sections describe how to do this.
+
+[!include[Common content for creating extensions](~/concepts//messaging-extensions/messaging-extensions-common.md)]
+
+### Action type message extensions
+
+To initiate actions from a  messaging extension set the `type` parameter to `action`. Below is an example of a manifest with a search and a create command. A single messaging extension can have up to 10 different commands associated with it. This can include both multiple search and multiple action-based commands.
```

Comment:
I think you can safely delete "associated with it".

Reply:
Done